lib/src/api/player/player_api.dart

```diff
@@ -86,4 +86,19 @@ abstract class PlayerApi {
 
   /// Provides access to Analytics related functionality.
   AnalyticsApi get analytics;
+
+  /// Whether casting to a cast-compatible remote device is available.
+  /// `CastAvailableEvent` signals when casting becomes available.
```

Comment:
Let's not forget to use square bracket notation once the events exits 👀

Reply:
Done: 8e5b06961ac4c8a380158b8f5712af8fa662f550